operating/manage-vulnerabilities/scan-rhcos-node-host.adoc

```diff
@@ -7,12 +7,15 @@ include::modules/common-attributes.adoc[]
 toc::[]
 
 [role="_abstract"]
-For {ocp}, {op-system-first} is the only supported operating system for control plane. Whereas, for node hosts, {ocp} supports both {op-system} and {op-system-base-full}.
+For {ocp}, {op-system-first} is the only supported operating system for control plane. For node hosts, {ocp} supports both {op-system} and {op-system-base-full}.
 With {rh-rhacs-first}, you can scan {op-system} nodes for vulnerabilities and detect potential security threats.
 
 {product-title-short} scans {op-system} RPMs installed on the node host, as part of the {op-system} installation, for any known vulnerabilities.
 
-First, {product-title-short} analyzes and detects {op-system} components. Then it matches vulnerabilities for identified components by using {op-system-base} and OpenShift 4.X Open Vulnerability and Assessment Language (OVAL) v2 security data streams.
+First, {product-title-short} analyzes and detects {op-system} components. Then it matches vulnerabilities for identified components by using {op-system-base} and the following data streams:
+
+* OpenShift 4.X Open Vulnerability and Assessment Language (OVAL) v2 security data streams, if the StackRox scanner is used for node scanning
```

Comment:
StackRox Scanner is a proper noun

the StackRox scanner -> StackRox Scanner

Reply:
Don't forget ^^